releases/release-1.31/release-notes/maps/pr-124038-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 124038
+releasenote:
+  text: 'kubeadm: stopped storing the ResolverConfig in the global KubeletConfiguration
+    and instead set it dynamically for each node'
```

Comment:
```suggestion
    and now sets it dynamically for each node.
```

Reply:
Based on documentation past tense is preferable not sure if we should stick with "instead set" or "now sets"